terraform/variables.tf

```diff
@@ -142,9 +142,9 @@ variable "domain" {
 }
 
 variable "statuscake_ssl_contact_group" {
```

Comment:
Can it be renamed to statuscake_ssl_contact_groups? as it contains multiple groups

Reply:
Fixed this .